### PR TITLE
add description of odd characters 

### LINKED ISSUE #12: samples/sample1 includes odd character sequences
`samples/sample1.json` contains character sequences, such as `-lrb-` and `-rrb-`, `@highlight`,  and `__...__`
Do they have any significance? If they don't, we probably shouldn't include them. If they do and they impact the result we need to explain them.

### PR DESCRIPTION
**Pull request by autoih**
close https://github.com/IBM/MAX-Text-Summarizer/issues/12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,6 +133,8 @@ You should see a JSON response like that below:
 }
 ```
 
+As you can see, the text summarizer not only gives the summary text but also filters out some special characters such as `-lrb- `, `-rrb-` and `@highlight`, etc, from the [Daily Mail](https://github.com/abisee/cnn-dailymail) dataset.
```
Comment (xuhdev) [inline README.md]:
```suggestion
In this example, the text summarizer not only gives the summary text but also filters out some special characters such as `-lrb-` (representing `(`), `-rrb-` (representing `)`), `@highlight`, etc, from the [Daily Mail](https://github.com/abisee/cnn-dailymail) dataset.
```

Comment (autoih) [reply README.md]: Thanks @xuhdev. I'd use `comparing to...` to show the difference.

File: README.md [hunk 2]
```diff
@@ -133,6 +133,8 @@ You should see a JSON response like that below:
 }
 ```
 
+Comparing to the original [sample email](samples/sample1.json), the text summarizer not only gives the summary text but also filters out some special characters such as `-lrb-` (representing `(`), `-rrb-` (representing `)`), etc, from the [Daily Mail](https://github.com/abisee/cnn-dailymail) dataset.
```
Comment (xuhdev) [inline README.md]:
```suggestion
The text summarizer preserves in the output summary text some special characters such as `-lrb-` (representing `(`), `-rrb-` (representing `)`), etc. that appear in the input [sample](samples/sample1.json), which is excerpted from the [Daily Mail](https://github.com/abisee/cnn-dailymail) dataset.
```

Comment (autoih) [reply README.md]: Thanks @xuhdev.  Hi @MLnick, may I also have your suggestions? Thanks.

Comment (MLnick) [reply README.md]: This seems fine

**Review by MLnick — APPROVED** [review]

**Pull request merged**